### PR TITLE
CM-230: fill json_ext when creating tasks

### PR DESCRIPTION
**Pull request by jdolkowski**
https://openimis.atlassian.net/browse/CM-230

This PR provides changes needed for displaying data for tasks on frontend.

### CODE REVIEW

File: tasks_management/services.py [hunk 1]
```diff
@@ -181,6 +183,9 @@ def _adjust_create_task_data(self, obj_data):
                 obj_data[key] = str(obj_data[key])
         return obj_data
 
+    def _data_for_json_ext_create(self, obj_data):
```
Comment (sniedzielski) [inline tasks_management/services.py:186]: Do we need to pass 'obj_data' argument in method?

File: tasks_management/gql_queries.py [hunk 2]
```diff
@@ -69,11 +69,14 @@ class Meta:
         connection_class = ExtendedConnection
 
     def resolve_current_entity_data(self, info):
-        entity = BenefitPlan.objects.first()
+        entity = self.entity
         if entity:
             serialized_json = _convert_to_serializable_json(entity)
             return serialized_json
-        return {}
+        return "{}"
```
Comment (sniedzielski) [inline tasks_management/gql_queries.py]: Please create constant either outside of classes or in that method. Would be more readable.

File: tasks_management/services.py [hunk 3]
```diff
@@ -228,6 +234,9 @@ def _adjust_update_task_data(self, obj_data):
                 obj_data[key] = str(obj_data[key])
         return obj_data
 
+    def _data_for_json_ext_update(self, obj_data):
```
Comment (sniedzielski) [inline tasks_management/services.py:237]: Same as on line 188. Moreover that method could be unified to have one signature both for create/update context. Seems that method behaves in the same way for both contexts.

File: tasks_management/services.py [hunk 4]
```diff
@@ -275,6 +285,9 @@ def _adjust_delete_task_data(self, obj_data):
                 obj_data[key] = str(obj_data[key])
         return obj_data
 
+    def _data_for_json_ext_delete(self, obj_data):
```
Comment (sniedzielski) [inline tasks_management/services.py:288]: Same as previous raised comments with 'context' and potential redundant 'argument'

**Review by sniedzielski — COMMENTED** [review]
I left minor comments with suggestions to little improvements. Please refer to them @jdolkowski

File: tasks_management/services.py (hunk 1 above)
Comment (jdolkowski) [reply tasks_management/services.py:186]: removed

File: tasks_management/gql_queries.py (hunk 2 above)
Comment (jdolkowski) [reply tasks_management/gql_queries.py]: done

File: tasks_management/services.py (hunk 1 above)
Comment (jdolkowski) [reply tasks_management/services.py:186]: can't remove because it is being called with the argument in task_create

Comment (sniedzielski) [reply tasks_management/services.py:186]: ok

**Review by sniedzielski — APPROVED** [review]

**Pull request merged**